src/components/Play/Play.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+
+import {noop} from '@shopify/javascript-utilities/other';
+import {ReactComponent} from '@shopify/react-utilities/types';
+import compose from '@shopify/react-compose';
+
+import {WithContextTypes} from '../../types';
+import withContext from '../WithContext';
+
+import Popover, {Props as PopoverProps} from '../Popover';
+import Button, {Props as ButtonProps} from '../Button';
+
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+class ToggleState extends React.PureComponent<
+  {children: React.ReactNode},
+  ToggleStateState
+> {
+  state: ToggleStateState = {
+    active: false,
+  };
+
+  get getContext(): ToggleStateContext {
+    return {
+      active: this.state.active,
+      toggleState: this.toggleState,
+    };
+  }
+
+  render() {
+    return (
+      <ToggleStateProvider value={this.getContext}>
+        {this.props.children}
+      </ToggleStateProvider>
+    );
+  }
+
+  toggleState = () => {
+    const {active} = this.state;
+    this.setState({active: !active});
+  };
+}
+
+const {
+  Provider: ToggleStateProvider,
+  Consumer: ToggleStateConsumer,
+} = React.createContext<ToggleStateContext>({
+  active: false,
+  toggleState: noop,
+});
+
+interface ToggleStateState {
+  active: boolean;
+}
+
+interface ToggleStateContext extends ToggleStateState {
+  toggleState(): void;
+}
+
+interface PlayPopoverProps extends Omit<PopoverProps, 'active' | 'onClose'> {
+  initialActive?: boolean;
+}
```

Comment:
Here we're pulling off the `active` and `onClose` props since they are provided magically through context. For convenience, we're also adding an `initialActive` state so you could configure the Popover as open if you want.